docs/Encyclopedia/pages/GATK4_MuTect2_Tumor-Only.md

```diff
@@ -0,0 +1,29 @@
+# GATK4 MuTect2 Tumor-Only
+
+## Description ##
+
+GATK4 MuTect2 Tumor-Only is a somatic variant calling pipeline utilized in the GDC for analyzing tumor-only whole exome sequencing (WXS) and targeted sequencing data. 
```

Comment:
add "without normal samples" to the end of this sentence.

Reply:
added